resources/views/account/partials/shared-wishlist.blade.php

```diff
@@ -0,0 +1,3 @@
+<a v-if="wishlist.shared" :href="'{{ route('wishlist.shared', '') }}/' + wishlist.sharing_token" class="text-ct-inactive text-sm hover:underline">
+    @lang('(shared)')
```

Comment:
I'd place the parentheses outside of the @lang function, so it can be used more broadly